### PR TITLE
Adjust attachment variables for TWA templates

### PR DESCRIPTION
**Pull request by norkans7**
matches the attachments for TWA channel as for #752

### CODE REVIEW

**Review by rowanseymour — APPROVED** [review]

File: handlers/twiml/handlers.go [hunk 1]
```diff
@@ -253,7 +253,16 @@ func (h *handler) Send(ctx context.Context, msg courier.MsgOut, res *courier.Sen
 
 		for _, comp := range msg.Templating().Components {
 			for varKey, varIndex := range comp.Variables {
-				contentVariables[varKey] = msg.Templating().Variables[varIndex].Value
+				value := msg.Templating().Variables[varIndex].Value
+				attType, value := handlers.SplitAttachment(value)
+				attType = strings.Split(attType, "/")[0]
+
+				// not attachment ignore the split, revert to original entire value
```
Comment (norkans7) [inline handlers/twiml/handlers.go:260]: @rowanseymour hold on I found a better way here

File: handlers/twiml/handlers.go [hunk 2]
```diff
@@ -253,7 +253,16 @@ func (h *handler) Send(ctx context.Context, msg courier.MsgOut, res *courier.Sen
...
+				if attType != "image" && attType != "video" && attType != "application" {
+					value = msg.Templating().Variables[varIndex].Value
+				}
```
Comment (rowanseymour) [inline handlers/twiml/handlers.go:263]: actually this feels a bit weird and is liable to do the wrong thing if a text variable looks like an attachment... can't we check variable type like in the WhatsApp handler ?

**Review by rowanseymour — CHANGES_REQUESTED** [review]

File: handlers/twiml/handlers.go (hunk 1 above)
Comment (norkans7) [reply handlers/twiml/handlers.go:260]: checking the type `msg.Templating().Variables[varIndex].Type` so we can split as attachment

File: handlers/twiml/handlers.go (hunk 2 above)
Comment (norkans7) [reply handlers/twiml/handlers.go:260]: Yes, I had found that just after pushing

**Review by rowanseymour — APPROVED** [review]

**Pull request merged**